mlflow/openai/autolog.py

```diff
@@ -360,6 +363,60 @@ def _process_last_chunk(span: LiveSpan, chunk: Any, inputs: dict[str, Any], outp
     _end_span_on_success(span, inputs, output)
 
 
+def _reconstruct_completion_from_stream(chunks: list[Any]) -> Any:
+    """
+    Reconstruct a completion object from streaming chunks.
+
+    This preserves the structure and metadata that would be present in a non-streaming
+    completion response, including ID, model, timestamps, usage, etc.
+    """
+    if not chunks:
+        return None
+
+    if chunks[0].object == "text_completion":
+        # Handling for the deprecated Completions API. Keep the legacy behavior for now.
+        def _extract_content(chunk: Any) -> str:
+            return chunk.choices[0].text if chunk.choices else ""
+
+        return "".join(_extract_content(chunk) or "" for chunk in chunks)
```

Comment:
```suggestion
        # Handling for the deprecated Completions API. Keep the legacy behavior for now.
        return "".join(c.choices[0].text if c.choices else "" for chunk in chunks)
```

do we need this function?